### ISSUE TITLE
Version 3.3.0 is breaking the action.

### ISSUE THREAD

**Issue by TomerFi** [axel-op/googlejavaformat-action#6]
Version [3.3.0](https://github.com/axel-op/googlejavaformat-action/releases/tag/v3.3.0) breaks the action.

My configuration was:
```yaml
      - name: Check java style format
        uses: axel-op/googlejavaformat-action@v3
        with:
          skipCommit: true
```

Which worked well until the recent 3.3.0 release.
Now it fails and throws:
```text
Error: Parameter token or opts.auth is required
```

Pinning version [3.2.0](https://github.com/axel-op/googlejavaformat-action/releases/tag/v3.2.0), fixed the issue for me:

```yaml
      - name: Check java style format
        uses: axel-op/googlejavaformat-action@v3.2.0
        with:
          skipCommit: true
```

**Comment by axel-op** [comment]
Thanks for notifying me, I should have performed more extensive tests. This should be fixed now with [`v3.3.1`](https://github.com/axel-op/googlejavaformat-action/releases/tag/v3.3.1).

**Comment by axel-op** [comment]
There were actually multiple bugs with this version, my bad. I fixed them in [`v3.3.2`](https://github.com/axel-op/googlejavaformat-action/releases/tag/v3.3.2). Sorry for this!

**Comment by TomerFi** [comment]
I can confirm the issue has been fixed after bumping to `v3.3.2`
Thank you very much for this great action and the fast response!
